### ISSUE TITLE
Infowindow only shows first feature if multiple features with same ID is selected

### ISSUE THREAD

**Issue by tonnyandersson** [origo-map/origo#1444]
**Describe the bug**
Infowindow only shows first feature if multiple features with same feature ID number is selected.

**To Reproduce**
1. In default config, use `"infowindow": "infowindow"`
2. Duplicate the `origo-cities` layer and rename both like `origo-cities-1` and `origo-cities-2`
3. Load map and click on any feature

**Expected behavior**
Infowindow should display (at least) two hits from two different layers, but only one hit (the first) is displayed.

![image](https://user-images.githubusercontent.com/15725608/151541799-cdc241fe-9ffc-469f-9220-e2e19767d8b7.png)

Apparently, this is a known issue since it is mentioned in the code. The quick and dirty way to fix it would be to concatenate layer name and feature ID in the `some` test below (from _selectionmanager.js_). Any other suggestions?

``` javascript
function alreadyExists(item) {
  // FIXME: Take into consideration which layer? Also affects remove and all other by id functions.
  // Right now, if several layers use the same source, a feature can only be selected in one layer (the first attempted)
  return selectedItems.getArray().some((i) => item.getId() === i.getId());
}
```

**Comment by jokd** [comment]
I think adding the layer in the alreadyExists test will do it:
return selectedItems.getArray().some((i) => {item.getId() === i.getId() && item.selectionGroup === i.selectionGroup });

**Comment by steff-o** [comment]
I wrote that comment when I was fixing something else. At that time it was outside the scope of my issue and I probably did not bother to write an issue as I did not know if it actually could happen, it just looked wrong.

I think there are several places in the code that it is assumed that each feature has a unique id. For wfs if probably works unless there are several layers using the same source layer as in WFS layers the _id_ is `<Layername>.<PK>` (At least in Geoserver). But for AGS the _id_ is just `<PK>`. 

If I am not mistaken, there is also an issue with this in the editstore, but it is probably less likely that there are two editable layers on the same source.
